comp/api/api/apiimpl/internal/config/endpoint.go

```diff
@@ -12,17 +12,28 @@ import (
 	"fmt"
 	"html"
 	"net/http"
+	"strings"
 
 	gorilla "github.com/gorilla/mux"
 
 	"github.com/DataDog/datadog-agent/pkg/config"
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
+const prefixPathSuffix string = "."
+
 type authorizedSet map[string]struct{}
 
-var authorizedConfigPathsCore = authorizedSet{
-	"api_key": {},
+var authorizedConfigPathsCore = buildAuthorizedSet(
+	"api_key", "site", "dd_url", "logs_config.dd_url", "ha",
+)
+
+func buildAuthorizedSet(paths ...string) authorizedSet {
+	authorizedPaths := make(authorizedSet)
```

Comment:
nit: Since we know the capacity, of the map should we use it when initailizing the map? 
`authorizedPaths := make(authorizedSet, len(paths))`

Reply:
Yeah, that's reasonable.

Addressed in d2b460e188b31a6555bc556667f56c2ff326b190.